packages/website/src/types/common.ts

```diff
@@ -29,3 +71,23 @@ export type CharacterDetail={
     collects: number
   }
 }
+
+export type Subject={
```

Comment:
```suggestion
export type Subject = {
```
lint 没报错吗？

NOTE to me: lint 可能需要调整下